GiniVision/Classes/Core/MultipageReviewViewController.swift

```diff
@@ -226,6 +239,12 @@ extension MultipageReviewViewController {
                                          scrollPosition: .centeredHorizontally)
         self.collectionView(self.pagesCollection, didSelectItemAt: indexPath)
     }
+    
+    func reloadCollections() {
+        self.mainCollection.reloadData()
+        self.pagesCollection.reloadData()
+        self.selectItem(at: 0)
```

Comment:
What if the collections are empty?

Reply:
This is already fixed in another PR.

Reply:
Fixed in another PR? What does that mean? Why isn't it fixed here?

Reply:
There is a PR depending on this one.